Bottom: 99919cfc6b51fef26e833b8ca215ccc2f95d2b8d
Top:    ede00b8f02796535a32c5d52d9688cd8df285613
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 20:14:09 +0100

Refresh of hid-gtk-extend-layer-selector-

---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 295440b..432b443 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -339,6 +339,7 @@ ghid_layer_selector_init (GHidLayerSelector *ls)
   ls->accel_group = gtk_accel_group_new ();
   ls->action_group = gtk_action_group_new ("LayerSelector");
   ls->n_actions = 0;
+
   for (i = 0; i < 20; ++i)
     ls->accel_available[i] = TRUE;
